Source/FullScreenPokemon.ts

```diff
@@ -3255,6 +3255,7 @@ module FullScreenPokemon {
                 for (j = 0; j < moves.length; j += 1) {
                     if (moves[j].title === thing.moveName) {
                         thing.moveCallback(player, partyPokemon[i]);
+                        return;
```

Comment:
When debugging I realized that after the move callback, the for loop is run a few more times (until every move is looked at) so it's more efficient this way.